change/@ni-nimble-components-37c84290-c5e8-4c31-a22e-e69547cc96c0.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "default buttons appearance updates (#663)",
```

Since this is going to impact every client that uses a button, it'd be good to be more specific. (Feel free to update if I got something wrong)
```suggestion
  "comment": "Default appearance for buttons better matches visual design spec. Text is now semi-bold, border is more prominent when focused, and border hover color is changed for dark theme",
```